client/src/reviews/ReviewsList.tsx

```diff
@@ -38,15 +38,17 @@ const ReviewsList: React.FC = () => {
 
   return (
     <div className='reviewList'>
-      <ReviewSort />
-      <div className='reviewTileContainer'>
-        {reviews
-          ? [...displayedReviews]
-          : null}
-      </div>
+      {reviews?.results.length
```

Comment:
If `reviews` is undefined then calling `?.results` will safely return `undefined` but then calling `.length` on that will fail.  We should probably optionally chain all the way down the call, or just get rid of the first one